crates/pallet-grandpa-finality-verifier/src/tests/justification.rs

```diff
@@ -0,0 +1,222 @@
+//! Utilities for testing runtime code.
+
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use crate::grandpa::GrandpaJustification;
+use crate::tests::keyring::{test_keyring, Account};
+use codec::Encode;
+use sp_application_crypto::TryFrom;
+use sp_finality_grandpa::{AuthorityId, AuthoritySignature, AuthorityWeight, SetId};
+use sp_runtime::traits::{Header as HeaderT, One, Zero};
+use sp_std::prelude::*;
+
+pub const TEST_GRANDPA_ROUND: u64 = 1;
```

Comment:
As a general rule I prefer `pub(crate)` or `pub(super)` if possible, then compiler often able to suggest which data structures are no longer used. If marked as `pub` it can remain in code for a long time after needed.

Reply:
ahh missed this one. Agree on crate or super :-D